recipes/con-duct/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "con-duct" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/con_duct-{{ version }}.tar.gz
+  sha256: 730062871f43aa6bbbe40db5149f178712605b2654c2d31d1c447747e850d168
+
+build:
+  # Package depends on terminal sessions, and does not currently support Windows
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=46.4.0
+    - pip
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - con_duct
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/con/duct/
+  summary: Runs a command and collects resource usage metrics
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - asmacdo
+    - yarikoptic
```

Comment:
@jwodder would you mind to be added/help if needed?

Reply:
Go ahead.